test/TestHelpers.jl

```diff
@@ -216,6 +216,8 @@ end
     @inbounds deleteat!(parent(A), first_idx:last_idx)
 end
 
+Base.resize!(A::OffsetVector, nl::Integer) = resize!(A.parent, nl)
```

Comment:
There's a bit of a conundrum here. I think `resize!(::OffsetVector, ::Integer)` makes sense if you're really thinking of these as arrays where indexing starts at 0 (for which I'd argue we'd be better off with a dedicated `ZArray` type). But OffsetArray is more general than that, since it has arbitrary start indices. Why should `resize!` affect the end index but not the start index?

On the other hand, since `OffsetArray` is an immutable `struct` and the offset is encoded as a field, there actually isn't a mechanism to alter the start index. So I guess this is OK, but it exposes a certain amount of awkwardness.